[NoSquash] Add App Privacy Policy (#279)
minetest/minetest.github.io: app-privacy-policy.md

* Game servers: third-party servers when playing online

Note: this policy only applies to the Minetest application, accessing the
mentioned services directly using a web browser is not covered.

### Table of contents

- [Version checking](#version-checking)
	- [Information collected](#information-collected)
	- [How it is used](#how-it-is-used)
- [Loading server list](#loading-server-list)
	- [Information collected](#information-collected-1)
	- [How it is used](#how-it-is-used-1)
	- [Who has access](#who-has-access)
	- [Period of retention](#period-of-retention)
	- [Location](#location)
- [Browsing Online Content (ContentDB)](#browsing-online-content-contentdb)
	- [Information collected](#information-collected-2)
	- [How it is used](#how-it-is-used-2)
	- [Who has access](#who-has-access-1)
	- [Period of retention](#period-of-retention-1)
	- [Location](#location-1)
	- [Removal requests](#removal-requests)
- [Online play](#online-play)
- [Third parties](#third-parties)
- [Future changes to privacy policy](#future-changes-to-privacy-policy)

## Version checking

### Information collected

When you open Minetest, it may contact www.minetest.net to fetch information

> about the most recent version. This HTTP request will contain the following
> information:

Reviewer: ```suggestion
about the most recent version. The following information will be transferred or included:
```

* IP address
* Minetest version
* Platform, OS, device, etc

### How it is used

> The version information is hosted on GitHub pages, we do not log requests.

Reviewer: ```suggestion
The version information is hosted on GitHub pages, which does not log requests.
```

Author: Do we know this?

Reviewer: Could always change it to "The website is hosted on GitHub pages, please refer to their privacy policy."

Author: > When a GitHub Pages site is visited, the visitor's IP address is logged and stored for security purposes, regardless of whether the visitor has signed into GitHub or not. For more information about GitHub's security practices, see [GitHub Privacy Statement](https://docs.github.com/en/site-policy/privacy-policies/github-privacy-statement).

## Loading server list

### Information collected

> When you open the Play Online tab, Minetest will request the server list. HTTP
> requests are logged with the following information:

Reviewer: ```suggestion
When you open the Play Online tab, Minetest will request the server list.
The following information will be transferred or included:
```

* IP address
* Minetest version
* Platform, OS, device, etc

### How it is used

> Logged HTTP requests are used to aid with debugging, development, and user
> support. They are also used to generate aggregated user statistics, see
> <https://kitsunemimi.pw/tmp/>.

Reviewer: ```suggestion
HTTP requests are logged to aid with debugging, development and user
support.
They may also be used to generate anonymized, aggregated user statistics.
```

### Who has access

sfan5 runs the server list and has access to the logs. The logs may be shared

> with others to aid in debugging, but care will be taken to remove any personal

Author: Yeah that's the key, PII should be removed. It's a clumsy sentence though

information.

> Aggregated statistics are shared publicly.

Reviewer: the intention here is good but it's not like we legally guarantee that any aggregated stats ever created are public

Author: Yeah, I removed this line from elsewhere should remove this one too

### Period of retention

> Logged HTTP requests are stored for 6 months.

Reviewer: let's say 3 or less

Author: ```suggestion
Logged HTTP requests are stored for 3 months.
```

### Location

The server list is currently located in the Netherlands.

> By using this service, you give permission for the data to be moved as needed.

Reviewer: the data the server list displays is a different matter but we're still talking about http logs with client requests, right?
there is no need ever to move the *logs* to anywhere else

Author: You may change the country the server is hosted in

When you open Minetest, it may contact www.minetest.net to fetch information
about the most recent version. The following information will be transferred or
included:

* IP address
* Minetest version
* Platform, OS, device, etc

### How it is used

The version information is hosted on GitHub Pages.
See [About GitHub Pages > Data collection](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#data-collection):

> When a GitHub Pages site is visited, the visitor's IP address is logged and
> stored for security purposes, regardless of whether the visitor has signed
> into GitHub or not. For more information about GitHub's security practices,
> see
> [GitHub Privacy Statement](https://docs.github.com/en/site-policy/privacy-policies/github-privacy-statement).

## Online play

Minetest allows you to play online on multiplayer game servers. Please note that
game servers are third-party, Minetest acts like a web browser connecting to a
website. Therefore, please refer to the game server's privacy policy.

When connecting to a game server, the following information is sent:

* IP address
* Minetest version
* Window size, display, and input information

> * Anything you do in-game (including chat messages)

Reviewer: could remove this section, it's likely to become out of date and we're shifting this responsibility anyway.

Author: As, we should remove the exact information and just mention to check the game server's privacy policy? I'm fine with that
